### DISCUSSION TITLE
measurement_express_vi

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#34] [Announcements]
# measurement_express_vi

什么是 Express VI

https://lv.qizhen.xyz/measurement_express_vi

**Comment by MisakaLee** [comment]
**测试程序相关的 Express VI**  的第四段内容重复了

**Reply by ruanqizhen** [reply]
多谢指正！我重新编辑了这段文字。

**Comment by XuY6969** [comment]
演示数据采集程序的例子参数不会设置，老报错

**Reply by ruanqizhen** [reply]
错误信息是什么？

**Reply by XuY6969** [reply]
大佬回我了呜呜，我现在会了，他提示 频率要小于等于采样率/2，默认采样率是1000Hz，根据你设置的600多Hz刚好超过采样率/2

**Reply by ruanqizhen** [reply]
嗯，我这里写的不清楚，我会修改一下的。
